src/components/OptionListContextProvider.tsx

```diff
@@ -139,10 +139,6 @@ function OptionsListContextProvider({children}: OptionsListProviderProps) {
     }, [personalDetails, reports]);
 
     const initializeOptions = useCallback(() => {
-        if (areOptionsInitialized.current) {
```

Comment:
Would this cause any performance impact?

Reply:
Good question! The way I see it is the following - this functionality was missing, and it needed to be added, so the app can function properly, it's not like we are compounding the performance of an existing functionality, we are adding one. 

Previously, these options would only be initialized on the very first render of the `MoneyRequestParticipantsList` and stay that way until you relog/restart, ignoring changes made to the reports/policies made by you and others.

Reply:
Yeah I don't have the best context on this one, so hence I am checking. May be @grgia could take a look as well.

Reply:
I don't have the full context on why we needed this initially